flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml

```diff
@@ -22,7 +22,13 @@
             <property name="flowNavigationOnSave" type="Boolean" role="inputOnly" label="Flow - Navigation on Save?" description="Check if you would like the flow to navigate to the Next screen (or Finish) when clicking Save.  For best results, hide the page footer in the flow screen."/>
             <property name="flowNavigationOnCancel" type="Boolean" role="inputOnly" label="Flow - Navigation on Cancel?" description="Check if you would like the flow to navigate to the Previous or Next screen when clicking Cancel.  For best results, hide the page footer in the flow screen."/>
             <property name="flowNavigationOnCancelDirection" type="String" role="inputOnly" label="Flow - Navigation Direction on Cancel (Previous or Next)" description="Specify Previous or Next for the flow navigation direction after the Cancel button is clicked."/>
+            <property name="showCancelButton" type="Boolean" label="Show Cancel Button?" description="This will hide cancel button if set FALSE"/>
```

Comment:
Some properties are so simple and obvious a description doesn't provide any new information